### ISSUE TITLE
Ages net_worthd

### ISSUE THREAD

**Issue by Badrgamalahmed** [Badrgamalahmed/mini-project-for-decision-tree-#1]
"%%writefile ages_net_worths.py

import numpy
import random

def ageNetWorthData():

    random.seed(42)
    numpy.random.seed(42)

    ages = []
    for ii in range(100):
        ages.append( random.randint(20,65) )
    net_worths = [ii * 6.25 + numpy.random.normal(scale=40.) for ii in ages]
### need massage list into a 2d numpy array to get it to work in LinearRegression
    ages       = numpy.reshape( numpy.array(ages), (len(ages), 1))
    net_worths = numpy.reshape( numpy.array(net_worths), (len(net_worths), 1))

    from sklearn.cross_validation import train_test_split
    ages_train, ages_test, net_worths_train, net_worths_test = train_test_split(ages, net_worths)

    return ages_train, ages_test, net_worths_train, net_worths_test"
 http://napitupulu-jon.appspot.com/posts/regression-ud.html#:~:text=%25%25writefile%20ages_net_worths.py%0A%0Aimport%20numpy,return%20ages_train%2C%20ages_test%2C%20net_worths_train%2C%20net_worths_test
